spec/integration/publisher_spec.rb

```diff
@@ -0,0 +1,24 @@
+require_relative "./i_spec.rb"
+
+RSpec.describe "Publishing a message" do
+  let(:publisher) { RailwayIpc::TestPublisher.instance }
+  let(:message)   { LearnIpc::Commands::TestMessage.new }
+  let(:encoded_message) { Base64.encode64(LearnIpc::Commands::TestMessage.encode(message)) }
+
```

Comment:
how do we feel about adding something like a todo `TODO: add testing once Sneakers GEM is updated` that we can than easily reference in a Jira ticket later?